### PR TITLE
[plugin.video.bfi] 1.0.0

### PR DESCRIPTION
**Pull request by FraserChapman**
### Description

This add-on enables playing of videos and movies from the BFI Player website. It is not created, maintained or in any way affiliated with the British Film Institute. The add-on only provides an interface to access the free content on the BFI Player website from Kodi.

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :

My first addon so please be gentle :)
Tried to follow best practice and code for python 2 and 3 
Any questions, issues, suggestions etc please let me know.

### CODE REVIEW

File: plugin.video.bfi/addon.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <import addon="script.module.routing" version="0.2.0"/>
+        <import addon="script.module.beautifulsoup4" version="4.3.2" />
+        <import addon="script.module.requests" version="2.12.4"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="main.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Search and play free movies from the BFI collection</summary>
+        <description lang="en">This add-on enables playing of videos and movies from the BFI Player website. It is not created, maintained or in any way affiliated with the British Film Institute. The add-on only provides an interface to access the free content on the BFI Player website from Kodi.</description>
```
Comment (enen92) [inline plugin.video.bfi/addon.xml]: en_GB as well

File: plugin.video.bfi/addon.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <language></language>
```
Comment (enen92) [inline plugin.video.bfi/addon.xml]: Add a few languages here. This is the language of the content, i.e. the movies

File: plugin.video.bfi/addon.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <platform>all</platform>
+        <license>MIT</license>
+        <forum></forum>
+        <website>www.fraserchapman.co.uk</website>
+        <email>fraser.chapman@gmail.com</email>
+        <source></source>
```
Comment (enen92) [inline plugin.video.bfi/addon.xml]: Please add a source (it is linked from the addon website and wiki) pointing to your github

File: plugin.video.bfi/main.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+from resources.lib import kodilogging
+from resources.lib import plugin
+
+import logging
```
Comment (enen92) [inline plugin.video.bfi/main.py]: Unused import

File: plugin.video.bfi/resources/lib/kodiutils.py [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+        "thumb": image.get("data-img-800", ""),
+        "fanart": image.get("data-img-1440", ""),
+        "poster": image.get("src")
+    } if "src" in image else {
+        "thumb": image,
+        "fanart": image,
+        "poster": image
+    })
+
+def icon(image):
```
Comment (enen92) [inline plugin.video.bfi/resources/lib/kodiutils.py]: You can use `xbmcaddon` to grab the icon. `xbmcaddon.Addon().getAddonInfo('icon')`

**Comment by enen92** [comment]
Thanks, merged

**Comment by FraserChapman** [comment]
Ace, thanks for catching those bits too!

**Pull request merged**